src/test/scala/io/github/sentenza/hacktoberfest18/util/ArrayUtilSpec.scala

```diff
@@ -11,5 +11,15 @@ class ArrayUtilSpec extends WordSpec with Matchers {
       val length = rand.nextInt(1000)
       arrayUtil.buildRandomArray(length).length shouldBe length
     }
+
+    "return an array whose values are capped by the length" in {
```

Comment:
Sweet! Inspecting and testing the elements was something we had to care about. ;)